pkg/kubelet/cm/topologymanager/scope_pod.go

```diff
@@ -79,6 +79,8 @@ func (s *podScope) accumulateProvidersHints(pod *v1.Pod) []map[string][]Topology
 
 	for _, provider := range s.hintProviders {
 		// Get the TopologyHints for a Pod from a provider.
+		// TODO (https://github.com/kubernetes/kubernetes/issues/130069):
+		// Plumb context to GetTopologyHints when migrating global klog to context logging
```

Comment:
What's global klog?

Reply:
I meant to say global logger https://github.com/search?q=repo%3Akubernetes%2Fkubernetes+%22v1+%5C%22k8s.io%2Fapi%2Fcore%2Fv1%5C%22%22&type=code.